k6LoadTests.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable */
+import { sleep } from 'k6';
+import http from 'k6/http';
+
+export const options = {
+  duration: '10m',
+  vus: 20,
```

Comment:
IMHO, this is a bit too low for a performance test.

Reply:
that was defined in spex